### PR TITLE
FIX - Failing build if downloading fig or TeamCity fails.

### PR DESCRIPTION
**Pull request by roberttaylor426**

### CODE REVIEW

File: service/Dockerfile [hunk 1]
```diff
@@ -4,10 +4,10 @@ FROM java:8
 EXPOSE 8111
 
 # This installs fig 1.0.1, which is needed for continuous deployment.
-RUN (curl -L "https://github.com/docker/fig/releases/download/1.0.1/fig-$(uname -s)-$(uname -m)" > /usr/local/bin/fig) && chmod +x /usr/local/bin/fig
+RUN (curl -L "https://github.com/docker/fig/releases/download/1.0.1/fig-$(uname -s)-$(uname -m)" > /usr/local/bin/fig) && chmod +x /usr/local/bin/fig --fail
```
Comment (SamirTalwar) [inline service/Dockerfile]: You've added the `--fail` to the `chmod` instruction, not the `curl` one.

Comment (SamirTalwar) [inline service/Dockerfile]: Also, I just realised we use `-o` in the other `curl` instruction and `>` in this one. Let's use the same in both.

Comment (roberttaylor426) [inline service/Dockerfile]: D'oh.

Comment (SamirTalwar) [inline service/Dockerfile]: I'd recommend putting the `--fail` directly after `curl` so it's obvious.

**Pull request merged**